ballerina/service_endpoint.bal

```diff
@@ -171,10 +171,12 @@ const decimal DEFAULT_LISTENER_TIMEOUT = 120; //2 mins
 # + secureSocket - The SSL configurations for the server endpoint
 # + timeout - Period of time in seconds that a connection waits for a read/write operation. Use value 0 to
 #                   disable the timeout
+# + maxInboundMessageSize - The maximum message size to be permitted for inbound messages
```

Comment:
Let's add the default size here.

Reply:
https://github.com/ballerina-platform/module-ballerina-grpc/pull/575/commits/09fd6c5e4f0cf3cddb17f0385abf9e833f5a332d